### ISSUE TITLE
Balancing issue: Copper in early game

### ISSUE THREAD

**Issue by SWeini** [pyanodon/PyBlock#11]
TLDR: Copper from ash/soot in early game is too slow

See my post from earlier in discord

The early game is still (too) slow, just calculated again with the current mod version.

For all of these calculations I'm totally ignoring landfill because it's kind of cheap in comparison.
And let's assume I use raw coal as fuel, because it's available just from Fawogae.
And let's assume I refuse to mine ore patches.

Looking at iron first:
1 Fawogae plantation needs 7 iron plates, and produces 10.5 iron plates per hour (60% of the Fawogae is needed to produce raw coal, only 40% is for iron ore), that's less than 1 hour until it has paid for itself. That is a good ratio. Wouldn't even need to speed up the game for it.

If you scale up, you need to account for the extra atomizers you need, 1 atomizer per 10 Fawogae plantations, that's actually another 2.5 required iron plates per plantation.

And if you really scale up you need 1 extra spore collector per 33 Fawogae plantations, that's another 4.3 iron plates.

And unless you want to be stuck handfeeding, the absolute minimum is 6 belts (8.2 iron plates) and 2 inserters (17 iron plates) per plantation. And I'm ignoring all the extra belts/inserters for collector/DDC/atomizer/furnace here.

So in total one Fawogae plantation needs 7 + 2.5 + 4.3 + 8.2 + 17 = 39 iron plates (not just 7) and it has paid off after 3.7 hours. Still somewhat acceptable for a seablock-style game.
Reaching copper from arum:
I believe we can all agree that copper from ash/soot is not scalable long-term and you need to reach Cadaveric arums as soon as possible.

What's the minimum amount of copper plates needed to start with arum?
1 offshore pump: 8.75
1 steam engine: 6
1 soil extractor: 55.75
1 spore collector: 33
1 fawogae plantation: 4
1 atomizer: 20
1 automated screener: 7.5
1 glassworks: 69.25
1 advanced foundry: 22.5
1 high pressure furnace: 41.75
1 botanical nursery: 83.25
6 arum codex: 32.5 x 6 = 195
1 lab: 49
264 automation science packs: 1.5 x 264 = 396

Total: 991.75
Starting inventory: 850

That's not good, so you need to make some copper from ash separation and you need at least
1 solid separator: 175

Adjusted total: 1166.75

Minimum production from ash separation: 316.75 copper plates, that's 31368 ash from 31368 raw coal from 105266 fawogae, that's 418 hours of a fawogae plantation working. Unless you want to wait 418 hours for your single fawogae plantation you need to scale up, I'd say at least to 20 plantations, that's still over 20 hours of production.
But here is a closer look at copper:
1 Fawogae plantation needs 4 copper plates, and produces 0.76 copper plates per hour, straight through raw coal. That's 5.3 hours until it has paid for itself.

But again we need to consider the whole cost:
1 plantation: 4
6 belts: 2.25
2 inserters: 3
1/33 spore collector: 1

Total: 10.25

So it actually has paid off only after 13.5 hours (ignoring cost of iron plates)

The fun thing: To scale up iron and copper early you should actually ignore science completely until you have made all the Fawogae plantations you want. That's the first automation science pack at 20 hours with not much to do before, except waiting.

The copper situation is why you can basically fuel everything from raw coal, because you need the ash, not the fuel value. Upgrading to coal/coke reduces the produced ash and thereby ash, that's not what we want. I haven't looked at logs->coal->ash, it's just a bit more science, but 85 copper plates per forestry, 60 per moss farm and 83 per botanical nursery is probably too much when copper is the bottleneck.

Summary: You need to scale up a bit, but it doesn't pay off until after several hours. Don't expect anyone to reach Cadaveric arum before 50 or even 100 hours. Or you rely on ore patches, which is not what Pyblock is about, right?

**Comment by Luaancz** [comment]
I think it's clear enough that you _are_ supposed to use the ore patches. That's why they've been added back. They're to bootstrap you before you can fully automate the "pure" processes. Using ore patches, I got to the first cadaveric arum at around 16 hours in my first game (mind, in an older version where the arum plantations didn't have the new very cheap costs).

And yes, the full automation cost for Fawogae plantations is pretty crazy. But... you can help yourself a lot by keeping busy early on. Using the same amount of resources, you can more than quadruple your production (even just for iron) by skipping the automation. When you build up the plantations themselves, you can start slowly adding the automation - first on the outputs, then on the inputs. Total time taken to double your outputs goes way down. Of course, it only makes sense if you're not running the game in the background :) And it's oh so satisfying to finally get _rid_ of the need to handfeed stuff after you've been doing it for a while :D 

It would be nice if it was more feasible to avoid the ore patches, sure. Copper is just the start of the troubles, too - tin makes everything about 4-6 times worse. Also, finding a raw coal patch really shows how heavily Py-alternative energy is balanced around loads of cheap raw coal. Maybe the thing that _really_ needs buffing is Fawogae farming? Or maybe the recipe that turns Fawogae into raw coal? On the other hand, it's really nice that it takes more Fawogae for fueling the atomizers than for the iron ore production itself - it makes for a satisfying bump in potential iron production when you switch to other fuel sources (maybe increase raw coal production but decrease raw coal's fuel value and coal gas/tar value?). Or, as silly as it would be in a normal pygame, just get more ash from raw coal burning, or outright get soot instead. Or just get more soot from ash. Soot really sucks :D
